test/meson.build

```diff
@@ -49,56 +49,188 @@ if install_tests
                                  unittestsdir / 'testdata')
 endif
 
-test_bootctl_json_sh = find_program('test-bootctl-json.sh')
-test_fstab_generator_sh = find_program('test-fstab-generator.sh')
-test_network_generator_conversion_sh = find_program('test-network-generator-conversion.sh')
-test_systemctl_enable_sh = find_program('test-systemctl-enable.sh')
-test_systemd_tmpfiles_py = find_program('test-systemd-tmpfiles.py')
-hwdb_test_sh = find_program('hwdb-test.sh')
+############################################################
+
+if conf.get('ENABLE_HWDB') == 1
+        hwdb_test_sh = find_program('hwdb-test.sh')
+        if want_tests != 'false'
```

Comment:
These double indentations could be avoided, no?

`if conf.get('ENABLE_HWDB') == 1 and want_tests != 'false'`